content/en/real_user_monitoring/reactnative/_index.md

```diff
@@ -0,0 +1,405 @@
+});
+```
+
+## Data Storage
+
+### Android
+
+Before data is uploaded to Datadog, it is stored in cleartext in your application's cache directory. This cache folder is protected by [Android's Application Sandbox][10], meaning that on most devices this data can't be read by other applications. However, if the mobile device is rooted, or someone tempers with the Linux kernel, the stored data might become readable.
+
+### iOS
+
+Before data is uploaded to Datadog, it is stored in cleartext in the cache directory (`Library/Caches`) of your [application sandbox][11], which can't be read by any other app installed on the device.
+
+## Development mode
+
+While in development mode, your application can submit extra events related to the React Native tooling, such as code transformation errors and requests to a local development server.
+
+To prevent these events from showing in the dashboard, you can disable errors and resources tracking in dev mode using the `__DEV__` flag:
+
+```
+const config = new DdSdkReactNativeConfiguration(
+	CLIENT_TOKEN,
+	ENVIRONMENT,
+	APPLICATION_ID,
+	true,
+	!__DEV__  /* trackResources will be false in DEV mode, true otherwise */,
+	!__DEV__  /* trackErrors will be false in DEV mode, true otherwise */,
+	trackingConsent
+)
+```
+
+## Troubleshooting
+
+### Usage with `use_frameworks!`
+
+If you have `use_frameworks!` enabled in your `Podfile`, running `pod install` after adding the SDK is likely to trigger an error like this one:
+
+```shell
+The 'Pods-MyApp' target has transitive dependencies that include statically linked binaries: (DatadogSDKBridge, DatadogSDKCrashReporting)
+```
+
+To prevent that error, you can overwrite `use_frameworks!` and install all pods as static except for the ones that needs to be a framework:
+
+```ruby
+dynamic_frameworks = ['DatadogSDKBridge','DatadogSDKCrashReporting']
+
+# Make all the other frameworks into static frameworks by overriding the static_framework? function to return true
+pre_install do |installer|
+  installer.pod_targets.each do |pod|
+    if !dynamic_frameworks.include?(pod.name)
+      def pod.static_framework?;
+        true
+      end
+      def pod.build_type;
+        Pod::BuildType.static_library
+      end
+    end
+  end
+end
+```
+
+**Note:** This solution comes from this [StackOverflow][14] post.
+
+## License
+
+For more information, see [Apache License, v2.0][9]
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: https://app.datadoghq.com/rum/application/create
+[2]: https://raw.githubusercontent.com/DataDog/dd-sdk-reactnative/main/docs/image_reactnative.png
+[3]: /account_management/api-app-keys/#api-keys
+[4]: /account_management/api-app-keys/#client-tokens
+[5]: https://github.com/wix/react-native-navigation
+[6]: https://www.npmjs.com/package/@datadog/mobile-react-native-navigation
+[7]: https://github.com/react-navigation/react-navigation
+[8]: https://www.npmjs.com/package/@datadog/mobile-react-navigation
```

Comment:
We can refer to the new integrated_libraries page here instead

Reply:
again, do you mean [8] should point to /real_user_monitoring/reactnative/integrated_libraries/?

Reply:
yes that's what I meant :)